content/en/admin/optional/tor.md

```diff
@@ -126,17 +128,17 @@ map $http_upgrade $connection_upgrade {
 server {
   listen 443 ssl http2;
   list [::]:443 ssl http2;
-  server_name mastodon.myhosting.com;
+  server_name mastodon.example.com;
   include /etc/nginx/snippets/mastodon.conf;
 
-  ssl_certificate /etc/letsencrypt/live/mastodon.myhosting.com/fullchain.pem;
-  ssl_certificate_key /etc/letsencrypt/live/mastodon.myhosting.com/privkey.pem;
+  ssl_certificate /etc/letsencrypt/live/mastodon.example.com/fullchain.pem;
+  ssl_certificate_key /etc/letsencrypt/live/mastodon.example.com/privkey.pem;
 }
 ```
 
 Replace the long hash provided here with your Tor domain located in the file at `/var/lib/tor/onion_service/hostname`.
```

Comment:
```suggestion
Replace the long hash provided here with your Tor domain located in the file at `/var/lib/tor/hidden_service/hostname`.
```

Reply:
Will need to do a cleanup + rebase over the next few days.